spec/features/openid_connect/openid_connect_spec.rb

```diff
@@ -56,11 +56,6 @@
       expect(current_path).to eq(openid_connect_authorize_path)
       expect(page).to have_content(t('openid_connect.authorization.errors.prompt_invalid'))
     end
-
-    it 'succeeds with a vtr param' do
```

Comment:
I do think that we want this coverage in the specs. I don't see in the new `vtr_spec` where we have coverage that includes inspected the decoded ID token or looking at the user_info response. For that reason I think we want this example.

Reply:
Put back.